pypeit/spectrographs/keck_deimos.py

```diff
@@ -293,6 +293,8 @@ def config_specific_par(self, scifile, inp_par=None):
             par['calibrations']['wavelengths']['method'] = 'full_template'
             par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_deimos_900ZD.fits'
             par['calibrations']['wavelengths']['lamps'] += ['CdI', 'ZnI', 'HgI']
+        # Arc lamps list from header
+        par['calibrations']['wavelengths']['use_header_lamps'] = True
```

Comment:
Can you add a brief comment or two somewhere
on this being the default for DEIMOS, 
e.g. https://pypeit.readthedocs.io/en/release/deimos.html ?